.scrutinizer.yml

```diff
@@ -19,31 +19,27 @@ build:
           file: build/coverage.xml
           format: clover
   nodes:
-    php72:
```

Comment:
These should remain if we keep php7.2 as a supported version.

Reply:
Fixed